app/src/controllers/step/actionCreators.js

```diff
@@ -52,12 +51,7 @@ export const includeInAutoAnalysisAction = defineGroupOperation(
   validateIncludeInAA,
 );
 
-export const unlinkIssueAction = defineGroupOperation(
-  NAMESPACE,
-  'unlink-issue',
-  unlinkIssue,
-  validateUnlinkIssue,
```

Comment:
Your changes break the validation process for an independent `unlink` operation.
The existing validation should be left as is.